packages/presentation_layer/lib/app/screens/palindrome/palindrome.dart

```diff
@@ -14,32 +14,31 @@ class Palindrome<D> extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(),
       body: StreamBuilder<BlocData>(
-        stream: bloc.dataStream,
-        initialData: BlocData.init(),
-        builder: (context, snapshot) {
-          const String verifyDefault = 'Result';
-          final BlocData? blocData = snapshot.data;
-          final PalindromeData? screenData = blocData?.data;
-          final String verifyResult = screenData is PalindromeData ? screenData.verifyInfo : verifyDefault;
+          stream: bloc.dataStream,
+          initialData: BlocData.init(),
+          builder: (context, snapshot) {
+            const String verifyDefault = 'Result';
+            final BlocData? blocData = snapshot.data;
+            final PalindromeData? screenData = blocData?.data;
+            final String verifyResult = screenData is PalindromeData
```

Comment:
you could be use like this if(screendata is Palindrome) {
return screenData.verifyInfo
}else{
verifyifyDefoult;
}
I mean left part of equal on the 23 stroke is unnecessary

it just may opinion!)

Reply:
ok!  Let see Sergey point of view.

Reply:
seems that u are speaking about 2 approaches, which do the same. both are ok